README.md

```diff
@@ -114,7 +114,7 @@ Specify the path to the cache location. Can be a file or a directory.
 type configType = 'flat' | 'eslintrc';
 ```
 
-- Default: `eslintrc`
+- Default: `flat`
```

Comment:
@ricardogobbosouza Just interesting, can we make it smarter - preload config and understand is it `flag` or  `eslintrc`? If no let's bump `eslint` in perr deps where flat config was introduce

Reply:
The Configuration File can be in several places and/or formats.

What can be done is to check the version of `eslint` used and if it is >= 9 define `flat`, otherwise `eslintrc`

Reply:
But I wouldn't recommend setting it automatically based on the `eslint` version.

Since this will be a major release, we're already enforcing the eslint default

Reply:
:+1: